### DISCUSSION TITLE
Is there a way to change the code blocks theme?

### DISCUSSION THREAD

**Discussion by vitorgouveia** [shuding/nextra#422] [Q&A]
I'm using the alpha version of Nextra and I ran into the Shiki docs and there I saw they have a lot of themes, just curious, is there a way to use other theme in the codeblocks?

**Comment by jermanuts** [comment]
Yes and no. [See](https://github.com/shuding/nextra/issues/788) changing `--nextra-primary-hue` will change codeblocks color, anchor links and both sidebar color. Example: https://docs.blink.sh/ and [their code](https://github.com/pcho/blink-theme/blob/de07e46364dd857d6b7801bd1ceb4635a74b2dee/src/styles.css#L13)

If you are wondering what ***deg is it's HSV that uses degrees than normal #FFFF hex code, example https://www.color-hex.com/color/ec04f0

**Comment by dsousa12** [comment]
My code blocks don't have any color.
Can help me?

**Comment by shuding** [comment]
Related docs: https://nextra.site/docs/guide/syntax-highlighting#customize-the-theme

**Comment by shawn-sandy** [comment]
Oops, wrong thread

**Comment by kilianbalaguer** [comment]
Im wondering how to do it too
bc i dont understand it
